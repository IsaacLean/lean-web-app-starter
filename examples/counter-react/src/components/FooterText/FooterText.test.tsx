import { render } from "@testing-library/react";

import FooterText from ".";

test("FooterText component matches snapshot", () => {
  const { asFragment } = render(<FooterText />);
  expect(asFragment()).toMatchSnapshot();
});
