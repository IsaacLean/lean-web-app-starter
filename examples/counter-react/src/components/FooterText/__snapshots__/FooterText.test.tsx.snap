// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FooterText component matches snapshot 1`] = `
<DocumentFragment>
  <footer
    class="text-center text-muted"
    id="footer-text"
  >
    <p
      class="mb-0"
    >
      Learn about the tech stack used for this project and more at its 
      <a
        class="text-muted"
        href="https://github.com/mattlean/lean-js-app-starter/tree/v1.0.0-rc/examples/counter-react"
      >
        GitHub repo
      </a>
      .
    </p>
    <p>
      Built with 
      <a
        class="text-muted"
        href="https://github.com/mattlean/lean-js-app-starter"
      >
        Lean JS App Starter
      </a>
      .
    </p>
  </footer>
</DocumentFragment>
`;
